cmd/examples/nse/nse.go

```diff
@@ -42,6 +44,10 @@ const (
 	EndpointSocketBaseDir = "/var/lib/networkservicemesh"
 	// EndpointSocket defines the name of NSM Endpoints operations socket
 	EndpointSocket = "nsm.endpoint.io.sock"
+
+	// NSMkeyNSELocalMechanism defines a key for nse local interface
+	// TODO (sbezverk) must be known globally
+	NSMkeyNSELocalMechanism = "nselocalmechansim"
```

Comment:
Because it sets up by NSM when it sends this info to NSE, so it has to be global and then it has to be consistent so NSE would know which key to use to retrieve this info.

Reply:
@sbezverk I would expect the interface_name to be set by the Dataplane, and just pass through the NSM.  Am I misunderstanding the info in this parameter, or is there a role for the NSM here I don't quite yet see?